allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb1_session_api.py

```diff
@@ -57,8 +58,9 @@ def __init__(self, path, *args, **kwargs):
             version_str = self._h5_root['nwb_version'][()]
             if not (version_str.startswith('NWB-1.') or version_str.startswith('1.')):
                 raise Exception('{} is not a valid NWB 1 file path'.format(self._path))
-        except:
-            raise Exception('{} is not a valid NWB 1 file path'.format(self._path))
+        except Exception:
+            warnings.warn(f"{self._path} is not a valid NWB 1 file path")
```

Comment:
This seems strange to me; warning, and then raise?

Reply:
The original code suppressed the error. I wanted to unsuppress that error but still provide a high-level explanatory message.

Reply:
warning is prob not necessary though